### PR TITLE
Fix transparent items not being rendered correctly against procedural skybox

### PR DESCRIPTION
**Pull request by huffman**

### CODE REVIEW

**Comment by huffman** [comment]
The changes in https://github.com/highfidelity/hifi/pull/8409/files causes the procedural skybox pipeline parameters to change. This removes those changes.

To test:
1. Go to playa
2. Launch the fireworks
3. Confirm that you can see them against the skybox
4. Do a general smoke test to make sure this doesn't introduce any render issues

**Comment by zzmp** [comment]
Won't this remove transparency for procedural entities?

**Comment by hifi-gustavo** [comment]
Build console output for commit: 1d2ea3be03728f64fb58a3319c3feb0fc2ed2874 available at: https://gist.github.com/2fc087a2d7a85c0b418b017a2894f051<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8439-1d2ea3b.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8439-1d2ea3b.dmg

**Comment by zzmp** [comment]
test this please

File: libraries/procedural/src/procedural/Procedural.cpp [hunk 1]
```diff
@@ -64,12 +64,6 @@ QJsonValue Procedural::getProceduralData(const QString& proceduralJson) {
 }
 
 Procedural::Procedural() {
-    _opaqueState->setCullMode(gpu::State::CULL_NONE);
-    _opaqueState->setDepthTest(true, true, gpu::LESS_EQUAL);
-    _opaqueState->setBlendFunction(false,
-        gpu::State::SRC_ALPHA, gpu::State::BLEND_OP_ADD, gpu::State::INV_SRC_ALPHA,
-        gpu::State::FACTOR_ALPHA, gpu::State::BLEND_OP_ADD, gpu::State::ONE);
-
     _transparentState->setCullMode(gpu::State::CULL_NONE);
```
Comment (zzmp) [inline libraries/procedural/src/procedural/Procedural.cpp]: You may be able to remove the setCullMode and setDepthTest for transparentState as well, if these are the defaults (I'm not sure that they are, you'd have to check).

**Comment by hifi-gustavo** [comment]
Build console output for commit: 1d2ea3be03728f64fb58a3319c3feb0fc2ed2874 available at: https://gist.github.com/2fc087a2d7a85c0b418b017a2894f051<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8439-1d2ea3b.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8439-1d2ea3b.dmg

**Comment by hevyw8** [comment]
Tested on DK2 and desktop both using an nvidia gtx 1060 card.

**fireworks are visible against skybox**

The only issue I noticed was some hue changes in the firework streak that only persisted during dusk and dawn in the skybox. see: [hue change](https://www.dropbox.com/s/ol0dc9n5het7j0z/Screenshot%20%2816%29.png?dl=0)

other screen caps:

[#8439](https://www.dropbox.com/sh/vhcfjxu0xob8s6i/AACXx7n6CY5b8FmSWMrSz2wNa?dl=0)

This is my first QA attempt. Any feedback on info provided and protocol would be appreciated. Thanks!

-Adan

**Pull request merged**